client/components/post-schedule/index.jsx

```diff
@@ -168,7 +169,8 @@ class PostSchedule extends Component {
 
 	render() {
 		return (
-			<div className="post-schedule" >
+			<div className="post-schedule">
+				{ this.props.site && <QuerySiteSettings siteId={ this.props.site.ID } /> }
```

Comment:
Right, I was thinking about that but once we have [a way to map `time_format` to a moment.js format](https://github.com/Automattic/wp-calypso/pull/10400/files#diff-df1fc8d02d6305e47e8598afe79e9df8), other child components of `PostSchedule` (like `Header`) will need it